template/src/components/Navbar.tsx

```diff
@@ -127,7 +128,7 @@ const Navbar = (props: any) => {
       <View
         style={[
           style.roomNameContainer,
-          Platform.OS === 'web' && !mobileAndTabletCheck()
+          Platform.OS === 'web' && !isMobileOrTablet()
```

Comment:
while we are at it. let's also update here with the utils

Reply:
Done